base/abstractdict.jl

```diff
@@ -701,3 +701,29 @@ function iterate(s::IdSet, state...)
     ((k, _), i) = y
     return (k, i)
 end
+
+"""
+    map!(f, values(dict::AbstractDict))
+Takes the function `f(value) and transforms values stored in `dict` with the transformation `value=f(value).
```

Comment:
```suggestion
Modifies `dict` by transforming each value from `val` to `f(val)`.
Note that the type of `dict` cannot be changed: if `f(val)` is not an instance of the key type
of `dict` then it will be converted to the key type if possible and otherwise raise an error.
```